ArcadiaTeamsBot/Controllers/BotController.cs

```diff
@@ -12,6 +11,7 @@
     public class BotController : ControllerBase
     {
         private readonly IBotFrameworkHttpAdapter botAdapter;
+

```

Comment:
And another code-style - it is allowed to not have empty line between single line fields definitions.

Reply:
Fixed